sdk/loadtesting/load-testing-rest/src/parameters.ts

```diff
@@ -12,260 +12,253 @@ import {
-  continuationToken?: string;
-  /** Prefix based, case sensitive search on searchable fields - description, executedUser. For example, to search for a test run, with description 500 VUs, the search parameter can be 500. */
+  /**
+   * Prefix based, case sensitive search on searchable fields - description,
+   * executedUser. For example, to search for a test run, with description 500 VUs,
+   * the search parameter can be 500.
+   */
   search?: string;
   /** Unique name of an existing load test. */
   testId?: string;
   /** Start DateTime(ISO 8601 literal format) of test-run execution time filter range. */
-  executionFrom?: Date | string;
+  executionFrom?: string;
   /** End DateTime(ISO 8601 literal format) of test-run execution time filter range. */
-  executionTo?: Date | string;
+  executionTo?: string;
   /** Comma separated list of test run status. */
   status?: string;
   /** Number of results in response. */
   maxpagesize?: number;
 }
 
-export interface TestRunListQueryParam {
-  queryParameters?: TestRunListQueryParamProperties;
+export interface ListTestRunsQueryParam {
+  queryParameters?: ListTestRunsQueryParamProperties;
 }
 
-export type TestRunListParameters = TestRunListQueryParam & RequestParameters;
-export type TestRunStopParameters = RequestParameters;
-export type TestRunListMetricNamespacesParameters = RequestParameters;
+export type ListTestRunsParameters = ListTestRunsQueryParam & RequestParameters;
+export type StopTestRunParameters = RequestParameters;
+export type ListMetricNamespacesParameters = RequestParameters;
 
-export interface TestRunListMetricDefinitionsQueryParamProperties {
+export interface ListMetricDefinitionsQueryParamProperties {
   /** Metric namespace to query metric definitions for. */
-  metricNamespace: string;
+  metricNamespace?: string;
 }
 
-export interface TestRunListMetricDefinitionsQueryParam {
-  queryParameters: TestRunListMetricDefinitionsQueryParamProperties;
+export interface ListMetricDefinitionsQueryParam {
+  queryParameters?: ListMetricDefinitionsQueryParamProperties;
 }
 
-export type TestRunListMetricDefinitionsParameters = TestRunListMetricDefinitionsQueryParam &
-  RequestParameters;
+export type ListMetricDefinitionsParameters = ListMetricDefinitionsQueryParam & RequestParameters;
 
-export interface TestRunListMetricsBodyParam {
+export interface ListMetricsBodyParam {
   /** Metric dimension filter */
-  body?: MetricRequestPayload;
+  body: MetricRequestPayload;
 }
 
-export interface TestRunListMetricsQueryParamProperties {
+export interface ListMetricsQueryParamProperties {
   /** The aggregation */
   aggregation?: string;
-  /** The interval (i.e. timegrain) of the query. */
-  interval?: "PT5S" | "PT10S" | "PT1M" | "PT5M" | "PT1H";
+  /**
+   * The interval (i.e. timegrain) of the query.
+   *
+   * Possible values: PT5S, PT10S, PT1M, PT5M, PT1H
+   */
+  interval?: string;
   /** Metric name */
-  metricname: string;
+  metricName?: string;
```

Comment:
Non-breaking, TSP and swagger inconsistancy.
[TODO] - Need to confirm why this happens?